sdk/storage/azure-storage-blob/src/test/java/com/azure/storage/blob/ContainerAPITest.groovy

```diff
@@ -1984,6 +1985,99 @@ class ContainerAPITest extends APISpec {
         response.getHeaders().getValue("x-ms-version") == "2017-11-09"
     }
 
+    def "create if not exists"() {
+        setup:
+        def cc = primaryBlobServiceClient.getBlobContainerClient(generateContainerName())
+
+        when:
+        cc.createIfNotExists()
+
+        then:
+        cc.exists()
+    }
+
+    def "create if not exists with response"() {
```

Comment:
You should take a look at the tests that exist for container create. You should be able to copy those almost exactly and then swap out the method being called.

Reply:
This will apply to pretty much all tests